### PR TITLE
Closes #52 Task controller

### LINKED ISSUE #52: Task controller

### PR DESCRIPTION
**Pull request by tarasbiletsky24**

### CODE REVIEW

File: LearnWithMentor/Controllers/TaskController.cs [hunk 1]
```diff
@@ -41,21 +43,22 @@ public TaskController(ITaskService taskService, IMessageService messageService,
...
                 }
-                return Request.CreateErrorResponse(HttpStatusCode.NoContent, "There are no tasks in database.");
+
+                return NoContent();
             }
             catch (Exception e)
             {
-
-                return Request.CreateErrorResponse(HttpStatusCode.InternalServerError, e);
+                ////tracer.Error(Request, ControllerContext.ControllerDescriptor.ControllerType.FullName, e);
```
Comment (OlexiiSamoyil) [inline LearnWithMentor/Controllers/TaskController.cs]: Why did you decide to leave this commented code?

Comment (tarasbiletsky24) [reply LearnWithMentor/Controllers/TaskController.cs]: to know where put future logger

Comment (OlexiiSamoyil) [reply LearnWithMentor/Controllers/TaskController.cs]: It's better to replace with ```//TODO: Log error here```
```suggestion
                //TODO: Log error here
```

**Comment by OlexiiSamoyil** [comment]
@tarasbiletsky24 you're so fast 😄

**Pull request merged**